crates/krator-derive/src/lib.rs

```diff
@@ -1,96 +1,104 @@
-//! A crate for deriving state machine traits in Kubelet. Right now this crate only consists of a
-//! derive macro for the `TransitionTo` trait. In addition to the `derive` attribute, this macro
+//! A crate for deriving state machine traits in Kubelet.
+//!
+//! Right now this crate consists of a [TransitionTo] derive macro for the `TransitionTo` trait.
+//!
+//! In addition to the `derive` attribute, this macro
 //! also requires the use of a custom attribute called `transition_to` that specifies the types that
 //! can be transitioned to. Not specifying this attribute will result in a compile time error.
-
+//!
+//! If the feature `admission-webhook` is enabled, this crate provides a [AdmissionWebhook] derive macro that
```

Comment:
This should probably contain the list of dependencies the generated code uses, which I think are `kube`, `k8s_openapi`, and `anyhow`.

Reply:
that makes sense, yes

Reply:
done in commit 193b6ac